src/lib/y2storage/md_level.rb

```diff
@@ -27,7 +27,30 @@ module Y2Storage
   # This is a wrapper for the Storage::MdLevel enum
   class MdLevel
     include StorageEnumWrapper
+    include Yast::I18n
+    extend Yast::I18n
 
     wrap_enum "MdLevel"
+
+    TRANSLATIONS = {
+      unknown: N_("Unknown"),
+      raid0:   N_("RAID0"),
+      raid1:   N_("RAID1"),
+      raid5:   N_("RAID5"),
+      raid6:   N_("RAID6"),
+      raid10:  N_("RAID10")
+    }
+    private_constant :TRANSLATIONS
+
+    # Returns human readable representation of enum.
+    # @return [String]
+    # @raise [RuntimeError] when called on unknown ennum value.
```

Comment:
Typo: ennum.

Since you have to change the documentation anyway to fix the typo, it would be cool if we clarify (is never too much clarification) that the resulting string would already be translated.

And talking about clarification... maybe we should also mention than an "unknown ennum value" means a value for which we don't have a i18n-enabled representation, which doesn't imply the value is invalid or unknown for Y2Storage/libstorage in general.